h2o-r/h2o-package/R/automl.R

```diff
@@ -16,6 +16,8 @@
 #'        validation_frame will be ignored.
 #' @param leaderboard_frame Leaderboard frame (H2OFrame or ID); Optional.  If provided, the Leaderboard will be scored using 
 #'        this data frame intead of using cross-validation metrics, which is the default.
+#' @param blending_frame Blending frame (H2OFrame or ID) used to train the Stacked Ensembles in the absence of cross-validated base models; Optional. 
```

Comment:
```suggestion
#' @param blending_frame Blending frame (H2OFrame or ID) used to train the the metalearning algorithm in Stacked Ensembles (instead of relying on cross-validated predicted values); Optional.
```